packages/daimo-expo-enclave/src/index.ts

```diff
@@ -39,6 +39,14 @@ export async function getBiometricSecurityLevel(): Promise<BiometricSecurityLeve
   return ExpoEnclaveModule.getBiometricSecurityLevel();
 }
 
+/**
+ * Forcibly use software (non-hardware backed) keys. Used for smooth operation on
+ * Android where unpredictable issues with hardware-backed keys are common.
```

Comment:
this makes it sound like we always use it on android

Reply:
fixed